dev/a11y_assessments/lib/main.dart

```diff
@@ -37,11 +37,46 @@ class App extends StatelessWidget {
       useCases.map((UseCase useCase) =>
           MapEntry<String, WidgetBuilder>(useCase.route, useCase.build)),
     );
+
     return MaterialApp(
-      title: 'Accessibility Assessments',
+      title: 'Accessibility Assessments Home Page',
       theme: lightTheme,
       darkTheme: darkTheme,
-      routes: <String, WidgetBuilder>{'/': (_) => const HomePage(), ...routes},
+      routes: <String, WidgetBuilder>{
+        '/': (_) => const AppWrapper(child: HomePage()),
+        ...routes
+      },
+    );
+  }
+}
+
+class AppWrapper extends StatelessWidget {
+
+  const AppWrapper({super.key, required this.child});
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return DynamicTitle(
```

Comment:
For home page it will use MaterialApp(title directly), I don't think you need to wrap the home page here?

Reply:
Updated based on your suggestions.